.circleci/config.yml

```diff
@@ -12,4 +13,8 @@ jobs:
           sudo curl -sLo consul.zip https://releases.hashicorp.com/consul/${CONSUL_VERSION}/consul_${CONSUL_VERSION}_linux_amd64.zip
           unzip consul.zip
           sudo cp consul /usr/local/bin/
+      - run: |
+          sudo curl -sLo vault.zip https://releases.hashicorp.com/vault/${VAULT_VERSION}/vault_${VAULT_VERSION}_linux_amd64.zip
```

Comment:
```suggestion
          curl -sLo vault.zip https://releases.hashicorp.com/vault/${VAULT_VERSION}/vault_${VAULT_VERSION}_linux_amd64.zip
```

Reply:
Yeah... sudo isn't needed on either of those curl calls. It seems to be a remnant of a previous, half-baked version of the curl install that spread. I'll eliminate that and force push up a fixed version in a few.

Thanks.